jobclient/python/cook/scheduler/client/jobs.py

```diff
@@ -112,197 +104,181 @@ class Job:
     command: str
     mem: float
     cpus: float
-
     uuid: UUID
     name: str
     max_retries: int
-    max_runtime: int
+    max_runtime: timedelta
     status: Status
     state: State
     priority: int
-    disable_mea_culpa_retries: bool
-
+    framework_id: str
+    retries_remaining: int
+    submit_time: datetime
+    user: str
     executor: Optional[Executor]
-    expected_runtime: Optional[int]
+    container: Optional[dict]
+    disable_mea_culpa_retries: Optional[bool]
+    expected_runtime: Optional[timedelta]
     pool: Optional[str]
     instances: Optional[List[Instance]]
     env: Optional[Dict[str, str]]
     uris: Optional[List[FetchableUri]]
-    container: Optional[dict]
     labels: Optional[Dict[str, str]]
     constraints: Optional[Set[Constraint]]
```

Comment:
Ah, this is where the `Constraint` class is used. I guess the question is: maybe we should wait to add this until we support specifying constraints on the job submission spec?

Reply:
Yeah, we could probably hold off on this and either leave it in raw list form or just ignore the parameter entirely